python/ray/autoscaler/autoscaler.py

```diff
@@ -54,6 +54,7 @@ class StandardAutoscaler:
     StandardAutoscaler is also used to bootstrap clusters (by adding workers
     until the target cluster size is met).
     """
+    provider = None
```

Comment:
Please do this in the constructor so we don't have a random class variable (and it's easier to track/see)

Reply:
thanks